server/routes/createInvite.ts

```diff
@@ -4,6 +4,16 @@ import { Response, NextFunction } from 'express';
 import lookupCommunityIsVisibleToUser from '../util/lookupCommunityIsVisibleToUser';
 import { SERVER_URL, SENDGRID_API_KEY } from '../config';
 
+export const Errors = {
```

Comment:
I really like this precedent to enumerate the Error types and then check for them in the tests.